packages/cli/src/commands/init.ts

```diff
@@ -201,7 +201,15 @@ const resolveOptionsWithPrompts = async (
     }
 
     if (!options.endpointSlug) {
-      resolvedOptions.endpointSlug = await promptEndpointSlug(path);
+      const resolvedPath = resolvePath(options.projectPath);
+      const packageJSONPath = pathModule.join(resolvedPath, "package.json");
+      const packageJSON = await readJSONFile(packageJSONPath);
+
+      if (packageJSON && packageJSON.endpointId) {
```

Comment:
Just realized that the `endpointId` needs to be looked for inside of `trigger.dev`. So `package['trigger.dev'].endpointId`

Reply:
Ah! 

So does that mean the `init` command of the CLI is only for trigger.dev? 

Not for any other project that uses it?